insights/insightsclient/insightsclient.go

```diff
@@ -146,8 +160,8 @@ func (c *Client) GetMimeType() string {
 
 // Send Posts source data to an endpoint
 func (c *Client) Send(ctx context.Context, endpoint string, source Source) error {
-	cv := c.clusterInfo.ClusterVersion()
-	if cv == nil {
+	cv, err := c.getClusterVersion()
+	if err != nil || cv == nil {
```

Comment:
Maybe it would be better to distinguish between the two situations (same as in the IO), but it's probably a minor detail.

Reply:
Created distinguishing error.

Reply:
Thanks Chris. Checking